share/dkg/dkg.go

```diff
@@ -701,9 +711,12 @@ func (d *DistKeyGenerator) ProcessResponses(bundles []*ResponseBundle) (*Result,
 	}
 
 	signature, err := d.sign(bundle)
+	if err != nil {
+		return nil, nil, err
+	}
 	bundle.Signature = signature
 	d.c.Info(fmt.Sprintf("%d justifications returned", len(justifications)))
-	return nil, bundle, err
+	return nil, bundle, nil
```

Comment:
I believe this would overwrite the named return from the defered call, no?

```suggestion
	res, err = nil, nil
	jb = bundle
	return
```

Reply:
Same with the `return nil, nil, nil` above, it would bypass the named return from the deferred call.

Reply:
It seems the named return from the defer has precedence:
https://go.dev/play/p/2BHqE6B1l5K